Documentation/operations/performance/tuning.rst

```diff
@@ -699,6 +699,29 @@ distributions offer kernel images with this option set or you can re-compile
 the Linux kernel. ``CONFIG_PREEMPT_NONE=y`` is the recommended setting for
 server workloads.
 
+Scaling
+=======
+
+By default, the `cilium daemonset <https://github.com/cilium/cilium/blob/main/install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml>`_
+is configured with an `inter-pod anti-affinity <https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity>`_
```

Comment:
nit: Prefer using block-level targets or anonymous links, see [docs style guide](https://docs.cilium.io/en/stable/contributing/docs/docsstyle/#links).

Reply:
Ah, much better. Thanks for the reference.